### ISSUE TITLE
Scrolling bug in iPhone Web Version

### ISSUE THREAD

**Issue by meconata** [FlutterFlow/flutterflow-issues#5105]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I have created a responsive version of a website, but I am experiencing an issue with scrolling on iOS devices. In the Safari and Chrome browsers on iPhones and iPads, it is impossible to scroll the website using a single touch (one finger). The website only scrolls if I use two fingers. However, on Android devices, the website works perfectly and can be scrolled using one finger without any issues. This problem seems to be specific to iOS devices.

### Expected Behavior

I believe scrolling should work as expected in all browsers, including Chrome and Safari, allowing users to scroll with one finger as they are accustomed to. This is the standard behavior that all users are familiar with. Due to this bug, we now have to explain to customers how to scroll on an iPhone versus how to scroll on an Android device, which is unnecessary and not user-friendly. This inconsistency is not acceptable and should be resolved.

### Steps to Reproduce

You can create a website yourself using a FlutterFlow project or open any existing FlutterFlow file, then publish it to any domain. Try scrolling the website on an iPhone using one finger—it is impossible. However, the same website works fine on Android devices, where you can scroll with one finger without any issues. Similarly, on an iPad browser, you will also find that scrolling with one finger is not possible. This issue seems specific to iOS devices.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT40hvHlvM91ofpE7qX2Kfpanjw5Jj9+UI0RicB7aCgofePtOol3evXsYVVpZMOrY3w6ekSmmk4x+NL8vfztJPcDFz+qbrZAybhUazz/RXGie5OBBoewZEdSP8NXf2q/17WJuiV7IshZZVob20yEefK+bDTbQ+/EImdISq/LZO4=

### Visual documentation

do not need visual documentation you can try by yourself or open my file "rentalestate"

### Environment

```markdown
- FlutterFlow version: 3.24.2 (released December 13, 2024)
- Platform: (Web, MacOS iPhone, iPad): Web
- Browser name and version: (web) Chrome, safari Latest versions
- Operating system and version affected: iPhone Version iOS 18.2
```

### Additional Information

I want to create websites in FlutterFlow or responsive admin dashboards for mobile apps, optimized for both phone and iPad versions. However, I am facing a problem where users have to scroll the page in their iPad or iPhone browsers using two fingers instead of the usual one-finger scrolling. This is not ideal, as it becomes inconvenient to constantly explain to customers how to scroll on their devices.

**Comment by zahamix** [comment]
Hey there, I'm having a similar issue on my side, do you have any updates?

**Comment by Joshua-57** [comment]
I have the same problem

**Comment by Alezanello** [comment]
Hello everyone,

Thank you for bringing this up. This is a known issue, and it's currently being addressed here: [GitHub Issue #4907](https://github.com/FlutterFlow/flutterflow-issues/issues/4907).

To keep things organized, I'll be closing this thread as a duplicate. Please feel free to follow the linked issue for updates.

Thank you for your understanding!

**Comment by meconata** [comment]
thanks again
